src/cljc/athens/parser/impl.cljc

```diff
@@ -24,7 +24,7 @@ block = (thematic-break /
          fenced-code-block /
          block-quote /
          paragraph-text)*
-thematic-break = #'[*_-]{3}'
+thematic-break = #'(\\*{3}(?!\\w[^\n]*(?<!\\s)\\*{3}(?!\\w))|[_-]{3})'
```

Comment:
That's a lot of regex.
I think we could drop support of `***` as thematic break.

Reply:
I agree, it makes more sense to do this. I can update the PR for this.